source/index-management.rst

```diff
@@ -405,20 +478,39 @@ Index View API
      * @note Drivers MAY opt to implement this method signature, the signature that
      *   takes an IndexModel as a parameter, or for those languages with method
      *   overloading MAY decide to implement both.
+     *
+     * @note Drivers MAY combine the two options types into a single one. If the options are
+     *   explicitly typed, the combined options type MUST be named DropOneIndexOptions or an acceptable
+     *   variation.
      */
-    dropOne(keys: Document, options: IndexOptions): Result;
+    dropOne(keys: Document, indexOptions: IndexOptions, options: Optional<DropOneIndexOptions>): Result;
```

Comment:
Do we know how many drivers implement this method or the next one? Go currently implements `dropOne(name, *DropIndexesOptions)` and it would be tough to add either of these two given no overloading.

Reply:
For context, I'm asking because the spec says drivers can implement the version that takes an `IndexModel`, which Go does, but also says "This is OPTIONAL until partial indexes are implemented", which already exist (https://docs.mongodb.com/manual/core/index-partial/) so it seems like something is out of date here.